src/Page/Object.elm

```diff
@@ -502,81 +535,125 @@ itemListView db selectedImpact results query =
+    [ tr []
         [ td [ class "ps-3 align-middle" ]
             [ div [ class "input-group", style "min-width" "180px" ]
                 [ input
                     [ type_ "number"
                     , class "form-control text-end"
-                    , amount |> Query.amountToFloat |> String.fromFloat |> value
-                    , step <|
-                        case process.unit of
-                            "kg" ->
-                                "0.01"
-
-                            "m3" ->
-                                "0.00001"
-
-                            _ ->
-                                "1"
+                    , quantity |> Query.quantityToInt |> String.fromInt |> value
+                    , step "1"
                     , onInput <|
                         \str ->
-                            case String.toFloat str of
-                                Just float ->
+                            case String.toInt str of
+                                -- FIX: don't update components based on their name
+                                -- swith to components ids as soon as they are implemented
+                                Just int ->
                                     UpdateItem
-                                        { amount = Query.amount float
-                                        , processId = process.id
+                                        { name = name
+                                        , quantity = Query.quantity int
+                                        , processes = processes |> List.map (\( amount, process ) -> { amount = amount, processId = process.id })
                                         }
 
                                 Nothing ->
                                     NoOp
                     ]
                     []
-                , span
-                    [ class "input-group-text justify-content-center fs-8"
-                    , style "width" "38px"
-                    ]
-                    [ text process.unit ]
                 ]
             ]
         , td [ class "align-middle text-truncate w-100" ]
-            [ text process.displayName ]
-        , td [ class "align-middle text-end" ]
-            [ Format.density process ]
+            [ text name ]
         , td [ class "text-end align-middle text-nowrap" ]
             [ Format.kg <| Simulator.extractMass itemResults ]
         , td [ class "text-end align-middle text-nowrap" ]
             [ Simulator.extractImpacts itemResults
                 |> Format.formatImpact selectedImpact
             ]
         , td [ class "pe-3 align-middle text-nowrap" ]
-            [ button [ class "btn btn-outline-secondary", onClick (RemoveItem process.id) ]
+            [ button [ class "btn btn-outline-secondary", onClick (RemoveComponent name) ]
                 [ Icon.trash ]
             ]
         ]
+    , tr []
+        [ td [ colspan 5 ]
+            [ details [ class "mb-2" ]
```

Comment:
Nit(ui): arrows should be aligned

![image](https://github.com/user-attachments/assets/796a3526-9bae-4a87-8d0b-1cc2509cffb9)

Reply:
It may also be worth aligning masses and impacts when processes are unfolded:
![image](https://github.com/user-attachments/assets/08710cab-8122-49bc-876a-d26be5616577)